site/docs/guides/kubernetes.md

```diff
@@ -175,13 +174,43 @@ database credentials from the secret that was created previously. We can do this
 ```yaml
 versionStoreType: JDBC
 jdbc:
-  jdbcUrl: jdbc:postgresql://postgres:5432/nessie
+  jdbcUrl: jdbc:postgresql://postgres:5432/nessiedb?currentSchema=nessie
+  secret:
+    name: postgres-creds
+    username: postgres_username
+    password: postgres_password
+```
+
+!!! note
+    The exact format of the JDBC URL may vary depending on the database you are using. Check the
+    documentation of your database and its JDBC driver for more information (for PostgreSQL, [check 
+    out this page](https://jdbc.postgresql.org/documentation/use/) and for MariaDB, [check out this 
+    one](https://mariadb.com/kb/en/about-mariadb-connector-j/#connection-strings)).
+
+The database and schema can be either passed in the JDBC URL (recommended) or as separate
+configuration options. The following configuration is equivalent to the one above:
```

Comment:
I believe for PosgreSQL _both_ the JDBC URL and the `schema` config variable must be set (to the same value). If the `schema` config is set, but not the URL default schema, Nessie will try to re-create tables on the _second_ run. I'm pretty sure I saw that in practice.

Reply:
Nessie will probably work fine with PosgreSQL if the schema is set in neither the JDBC URL, nor in the config.

Reply:
Hmm not sure, I just tried deployed a toy Nessie with this URL: `jdbc:postgresql://nessie-postgres-postgresql:5432/nessiedb?currentSchema=nessie` and no `schema`, and everything works as expected.

Then I tried `jdbc:postgresql://nessie-postgres-postgresql:5432/` with `catalog=nessiedb` and `schema=nessie2`, it works too.

As for the second run issue, I'm surprised given how the code proceeds:

https://github.com/projectnessie/nessie/blob/e9d7bebfd71deeec76ac9d0df661ca29c425a730/versioned/storage/jdbc/src/main/java/org/projectnessie/versioned/storage/jdbc/JdbcBackend.java#L178-L183

If you saw this happening it would be good to have a reproducer.

Reply:
Hmm. Actually I just saw something similar. I think the code above might be a bit off. I'm investigating.